README.md

```diff
@@ -93,6 +93,10 @@ Components that have been extracted to their own repository are still here as [g
 * [Scala](https://github.com/snowplow/snowplow-scala-tracker)
 * [Unity](https://github.com/snowplow/snowplow-unity-tracker)
 
+### TV
```

Comment:
```suggestion
#### TV
```

I also think we could probably put this above the Desktop & Server ones, I think it's more "platform-y" and sites nicer near the mobile trackers. I also wonder if Unity should be in something else too.

As I type this... I wonder if it's time to introduce a table here, it's getting a bit long!

|     Web    |    Mobile    | Gaming |  TV  | Desktop & Server |
|:----------:|:------------:|:------:|:----:|:----------------:|
| JavaScript | Android      | Unity  | Roku | Command line     |
| AMP        | iOS          |        |      | .NET             |
|            | React Native |        |      | Go               |
|            |              |        |      | Java             |
|            |              |        |      | Node.js          |
|            |              |        |      | PHP              |
|            |              |        |      | Python           |
|            |              |        |      | Ruby             |
|            |              |        |      | Scala            |

Reply:
I like the table idea! Just added it.